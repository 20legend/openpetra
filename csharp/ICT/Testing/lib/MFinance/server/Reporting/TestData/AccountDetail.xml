<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance/accountdetail.xml,Finance/accountdetailcommon.xml,Finance/finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />
  <Parameter id="MaxDisplayColumns" value="eInteger:6" />
  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eBoolean:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />
  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />
  <Parameter id="ColumnPosition" column="0" value="eDecimal:4618441417868443648" />
  <Parameter id="ColumnPositionIndented" column="0" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="0" value="eDecimal:4611686018427387904" />
  <Parameter id="ColumnPosition" column="1" value="eDecimal:4620693217682128896" />
  <Parameter id="ColumnPositionIndented" column="1" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="1" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="2" value="eDecimal:4622382067542392832" />
  <Parameter id="ColumnPositionIndented" column="2" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="2" value="eDecimal:4607182418800017408" />
  <Parameter id="ColumnPosition" column="3" value="eDecimal:4622945017495814144" />
  <Parameter id="ColumnPositionIndented" column="3" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="3" value="eDecimal:4619567317775286272" />
  <Parameter id="ColumnPosition" column="4" value="eDecimal:4626041242239631360" />
  <Parameter id="ColumnPositionIndented" column="4" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="4" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnPosition" column="5" value="eDecimal:4626885667169763328" />
  <Parameter id="ColumnPositionIndented" column="5" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="5" value="eDecimal:4613937818241073152" />
  <Parameter id="param_calculation" column="0" value="eString:Debit" />
  <Parameter id="param_calculation" column="1" value="eString:Credit" />
  <Parameter id="param_calculation" column="3" value="eString:Transaction Narrative" />
  <Parameter id="param_calculation" column="4" value="eString:Start Balance" />
  <Parameter id="param_calculation" column="5" value="eString:End Balance" />
  <Parameter id="param_ledger_number_i" value="43" />
  <Parameter id="param_with_analysis_attributes" value="eBoolean:false" />
  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
</Parameters>