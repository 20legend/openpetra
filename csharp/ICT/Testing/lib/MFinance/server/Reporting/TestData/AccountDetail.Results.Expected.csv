id,Details,CC,Acc,Debits,Credits,Narrative,Start Balance,End Balance
10300/0100/GB1/30/1/4,GB1 01/01/2016,10300,"0100",,-410.00,GB - Gift Batch 1,,
10300/0100/GB10/21/1/4,GB10 10/01/2016,10300,"0100",,-361.00,GB - Gift Batch 10,,
10300/0100/Detail,Sub Total:,,,0.00,-771.00,,0.00,"4,020.00"
10300/0100/Net Balance,Net Balance:,,,,-771.00,,,
10400/0100/GB4/27/1/6,GB4 04/01/2016,10400,"0100",,-71.00,GB - Gift Batch 4,,
10400/0100/GB5/26/1/6,GB5 05/01/2016,10400,"0100",,-212.00,GB - Gift Batch 5,,
10400/0100/GB10/21/1/5,GB10 10/01/2016,10400,"0100",,-376.00,GB - Gift Batch 10,,
10400/0100/GB12/19/1/4,GB12 12/01/2016,10400,"0100",,-181.00,GB - Gift Batch 12,,
10400/0100/GB14/17/1/5,GB14 14/01/2016,10400,"0100",,-21.00,GB - Gift Batch 14,,
10400/0100/GB16/15/1/5,GB16 16/01/2016,10400,"0100",,-68.00,GB - Gift Batch 16,,
10400/0100/GB21/10/1/3,GB21 21/01/2016,10400,"0100",,-450.00,GB - Gift Batch 21,,
10400/0100/GB23/8/1/7,GB23 23/01/2016,10400,"0100",,-25.00,GB - Gift Batch 23,,
10400/0100/GB25/6/1/7,GB25 25/01/2016,10400,"0100",,-113.00,GB - Gift Batch 25,,
10400/0100/GB28/3/1/4,GB28 28/01/2016,10400,"0100",,-280.00,GB - Gift Batch 28,,
10400/0100/GB30/1/1/4,GB30 30/01/2016,10400,"0100",,-172.00,GB - Gift Batch 30,,
10400/0100/Detail,Sub Total:,,,0.00,"-1,969.00",,0.00,"34,508.00"
10400/0100/Net Balance,Net Balance:,,,,"-1,969.00",,,
10500/0100/GB1/30/1/5,GB1 01/01/2016,10500,"0100",,-189.00,GB - Gift Batch 1,,
10500/0100/GB2/29/1/3,GB2 02/01/2016,10500,"0100",,-177.00,GB - Gift Batch 2,,
10500/0100/GB4/27/1/7,GB4 04/01/2016,10500,"0100",,-184.00,GB - Gift Batch 4,,
10500/0100/GB5/26/1/4,GB5 05/01/2016,10500,"0100",,-473.00,GB - Gift Batch 5,,
10500/0100/GB8/23/1/4,GB8 08/01/2016,10500,"0100",,-63.00,GB - Gift Batch 8,,
10500/0100/GB9/22/1/5,GB9 09/01/2016,10500,"0100",,-323.00,GB - Gift Batch 9,,
10500/0100/GB15/16/1/3,GB15 15/01/2016,10500,"0100",,-393.00,GB - Gift Batch 15,,
10500/0100/GB19/12/1/1,GB19 19/01/2016,10500,"0100",,-499.00,GB - Gift Batch 19,,
10500/0100/GB23/8/1/6,GB23 23/01/2016,10500,"0100",,-314.00,GB - Gift Batch 23,,
10500/0100/GB26/5/1/4,GB26 26/01/2016,10500,"0100",,-426.00,GB - Gift Batch 26,,
10500/0100/GB27/4/1/2,GB27 27/01/2016,10500,"0100",,-265.00,GB - Gift Batch 27,,
10500/0100/GB28/3/1/3,GB28 28/01/2016,10500,"0100",,-470.00,GB - Gift Batch 28,,
10500/0100/GB29/2/1/4,GB29 29/01/2016,10500,"0100",,-264.00,GB - Gift Batch 29,,
10500/0100/Detail,Sub Total:,,,0.00,"-4,040.00",,0.00,"68,700.00"
10500/0100/Net Balance,Net Balance:,,,,"-4,040.00",,,
"0100",Account Total:,,,0.00,"-6,780.00",,0.00,"107,228.00"
0100/Net Balance,Net Balance:,,,,"-6,780.00",,,
GrandTotal,Grand Total:,,,0.00,"-6,780.00",,0.00,"107,228.00"
GrandTotalNetBalance,Net Balance:,,,,"-6,780.00",,,
